coarse/models/errors.py

```
@@ -0,0 +1,13 @@
+from typing import List
+from pydantic import BaseModel
+from coarse.models.sections import SectionLabel
+
+class ErrorEntry(BaseModel):
+    INC: str
+    SEC: SectionLabel
+    CRT: str
+
+class Errors(BaseModel):
+    TYP: List[ErrorEntry]
+    GMR: List[ErrorEntry]
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)